src/app/core/services/issue.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 import { GithubService } from './github.service';
-import { catchError, exhaustMap, finalize, flatMap, map } from 'rxjs/operators';
+import { catchError, exhaustMap, finalize, flatMap, map, reduce } from 'rxjs/operators';
```

Comment:
I think we can remove `reduce` from the import here since it's no longer used 😄 .

Reply:
Thanks for the catch!